scripts/ci/selective_ci_checks.sh

```diff
@@ -475,8 +475,9 @@ function get_count_kubernetes_files() {
     echo
     local pattern_array=(
         "^airflow/kubernetes"
+        "^airflow/executors/kubernetes_executor.py"
```

Comment:
It's harmful here. That will exclude ""^airflow/executors/kubernetes_executor.py" from "core".

The "core" of airflow is defined as everything that is not defined in on of the selection criteria before:
```
COUNT_CORE_OTHER_CHANGED_FILES=$((COUNT_ALL_CHANGED_FILES - COUNT_WWW_CHANGED_FILES - COUNT_PROVIDERS_CHANGED_FILES - COUNT_CLI_CHANGED_FILES - COUNT_API_CHANGED_FILES - COUNT_KUBERNETES_CHANGED_FILES))
```

This means that any "airflow/executors/kubernetes_executor.py"  is treated as CORE/OTHER and it will trigger all tests (including kuberenetes tests). There are likely a number of unit tests depending on any change in "airflow/executors" package, so if we change anything in this package, we have to run all tests

On the other hand, if ONLY files in "kubernetest_tests" change - we should only run kubernetes tests.

There is another error here in fact  We shoud not add "^airflow/lkubernetes" here in the first place because there are some "kubernetes" unit tests (from "tests/kubernetes" folder) that could be skipped by accident if the "^airflow/kubernetes" only is changed).

Reply:
We should only live "chart" and "tests_kuberenetes" here. This will mean that when "charts" and "tests_kubernetes" to not change and no core changes the kubernetes tests will be skipped.